datahost-ld-openapi/test/tpximpact/test_helpers.clj

```diff
@@ -68,3 +68,8 @@
 
 (defn truncate-string [s n]
   (subs s 0 (min (count s) n)))
+
+(defn submap?
```

Comment:
I find it more useful to use the clojure `diff` function for this since it returns a structure representing the missing elements if they don't match, which makes debugging easier.

Reply:
Hmm. So I used diff on its own, but didn't like the fact that I had to repeat the `b` hash-map in tests when wanting to know if `b` was in `a`, in  all the places that `submap?` is used. I still prefer the clear intention of `submap?` too.

So I've swapped in `diff` to my implementation of `submap?`.  Doesn't really help with debugging, unless you were to set a breakpoint, which is what I'd do anyway.